webapp/i18n/en.json

```
@@ -7,6 +7,10 @@
   "jaas.embed-meeting-description": "(Experimental) When true, JaaS video is embedded as a floating window inside Mattermost by default.",
   "jaas.rsa-key": "RSA Private key for JaaS Authentication:",
   "jaas.rsa-key-description": "Specify your JaaS private key. You can get the private key by generating an API key pair from https://jaas.8x8.vc/#/apikeys OR from https://jaas.8x8.vc/#/start-guide in the \"Api Key\" section. (NOTE: While generating the key manually make sure to set the size to be 2048 and not 4096).",
+  "jaas.embed-enable": "true",
+  "jaas.embed-disable": "false",
```

There should only be two ids and these should be used everywhere.
```suggestion
  "jitsi.true": "true",
  "jitsi.false": "false",
```